README.md

```diff
@@ -1,173 +1,132 @@
+## About
+The Cloudinary Vue SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
+
+
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Vue SDK Guide](https://cloudinary.com/documentation/vue_integration).
+
+
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- Usage
+    - [Setup](#Setup)
+    - [Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Generating Image and HTML Tags](#Generate-Image-and-Video-HTML-Tags)
 
-Cloudinary is a [cloud-based service](https://cloudinary.com/solutions) that provides an end-to-end image and video management solution
-including uploads,storage, manipulations, optimizations and delivery. 
-All your media resources are optimized and delivered through a fast CDN using industry best practices.
+## Key Features
+- [Transform and Optimize Assets - Link to docs](https://cloudinary.com/documentation/vue_image_manipulation)
+- [Generate Image and Video Tags- Link to docs](https://cloudinary.com/documentation/vue_image_manipulation)
 
-Using Cloudinary's Vue.js SDK, you can easily perform smart image and video manipulations 
-using code that integrates seamlessly with your existing Vue.js application, and deliver optimized and responsive media to your users.
 
-This Readme provides basic installation and usage information. For the complete documentation, see the [Vue.js SDK guide](https://cloudinary.com/documentation/vue_integration)_
+## Version Support
+| SDK Version   | Vue 2    | Vue 3  |
+|---------------|----------|--------|
+| 1.0.0 - 1.2.3 | V        | X      |
 
-# 🛠️ Installation 
-## Vue 2.x
-1. Install using Vue-CLI
-    - After you create your application with Vue-CLI, navigate to the created app folder, and install Cloudinary SDK by:
-        ```bash
-        vue add cloudinary
-        ```
-    - Set up your cloudName and pick the components to use (or use all 😃)
-    ![Set up with cloudName and options](http://bit.ly/2WSKTf0)
-    - A `cloudinary.js` file will be added to your src directory, same level with your `main.js` file. Feel free to customize it if you like. And have fun with Cloudinary! 🤟
-    - More information on the plugin, check out [our Vue-CLI plugin for Cloudinary Repo](https://github.com/cloudinary/vue-cli-plugin-cloudinary)
 
-2. install using your favorite package manager (yarn, npm)
+
+## Installation
+### Install using Vue-CLI
+- After you create your application with Vue-CLI, navigate to the created app folder, and install Cloudinary SDK by:
     ```bash
-    npm install cloudinary-vue
-    yarn add cloudinary-vue
+    vue add cloudinary
     ```
-
+- Set up your cloudName and pick the components to use (or use all 😃)
+  ![Set up with cloudName and options](http://bit.ly/2WSKTf0)
```

Comment:
I love this little demo.  You should move it to the demo or cloudinary account though.  Currently it's in Maya Shavin's account!